client/js/components/user/CustomerSettings/CustomerSettings.vue

```diff
@@ -153,7 +157,8 @@
       <!-- Sign language video page -->
       <customer-settings-section
         v-if="hasPermission('field_sign_language_overview_video_edit')"
-        :title="Translator.trans('signLanguage.explanation')">
+        :title="Translator.trans('signLanguage.explanation')"
+        data-cy="customerSettings:overviewVideo">
```

Comment:
alphabetic order please 

please check the whole changes.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/3571/commits/78ca848b5899818a189ff165bec37fb2c45ba59f & https://github.com/demos-europe/demosplan-core/pull/3571/commits/3b68d63dad0c272e1750b3174dcda47ceab570bd
Thanks!